app/router/session_group.py

```diff
@@ -0,0 +1,36 @@
+from fastapi import APIRouter, HTTPException
+import requests
+from settings import settings
+
+router = APIRouter(prefix="/session-group", tags=["Session-Group"])
+session_group_db_url = settings.db_url + "/group-session/"
+
+
+@router.get("/{session_id}")
+def get_group_for_session(session_id: str):
+    """
+    This API returns group ID corresponding to the provided session ID, if the ID exists in the database
```

Comment:
Can a session be tagged only to a single group?

Reply:
I didn't get the question. A session is tagged to a single group only.